pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerMeter.java

```diff
@@ -40,6 +40,8 @@ public enum BrokerMeter implements AbstractMetrics.Meter {
   QUERY_VALIDATION_EXCEPTIONS("exceptions", false),
   // Query validation phase.
   UNKNOWN_COLUMN_EXCEPTIONS("exceptions", false),
+  // Queries preempted by accountant
+  QUERIES_PREEMPTED("query", true),
```

Comment:
Suggest changing to `QUERIES_KILLED` as that is more appropriate for this feature !!

Reply:
fixed